script.js

```diff
@@ -0,0 +1,336 @@
+class VirtualKeyboard {
+  constructor() {
+    this.elements = {
+      wrapper: null,
+      textarea: null,
+      keyboard: null,
+      keysContainer: null,
+      keys: [],
+      infoText: null,
+    };
+
+    this.props = {
+      lang: null,
+      textValue: '',
+      saveLang: 'eng',
+    };
+
+    this.keyLayout = {
```

Comment:
This configuration is better to move to another file, and pass to the class as a dependency. It will make you class more flexible.